### PR TITLE
[UA] 20211008: Travelers of the Multiverse

### PR DESCRIPTION
**Pull request by NickVendel**
- [X] - Astral Elf
- [X] - Autognome
- [X] - Giff
- [x] - Hadozee
- [x] - Plasmoid
- [X] - Thri-Kreen

### CODE REVIEW

File: unearthed-arcana/2021/20211008.xml [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+			<select type="Racial Trait" name="Spellcasting Ability (Astral Fire)" supports="UA20211008 Astral Fire" />
+		</rules>
+	</element>
+	<element name="Intelligence" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_ASTRAL_FIRE_INTELLIGENCE">
+		<supports>UA20211008 Astral Fire</supports>
+		<description>
+			<p>Your spellcasting ability for chosen cantrip is Intelligence.</p>
+		</description>
+		<sheet display="false">
+			<description>Your spellcasting ability for chosen cantrip is Intelligence..</description>
```
Comment (JayPea00101010) [inline unearthed-arcana/2021/20211008.xml:78]:
```suggestion
			<description>Your spellcasting ability for chosen cantrip is Intelligence.</description>
```

File: unearthed-arcana/2021/20211008.xml [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+			<stat name="ua:astral fire:spellcasting" value="Intelligence" inline="true" />
+		</rules>
+	</element>
+	<element name="Wisdom" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_ASTRAL_FIRE_WISDOM">
+		<supports>UA20211008 Astral Fire</supports>
+		<description>
+			<p>Your spellcasting ability for chosen cantrip is Wisdom.</p>
+		</description>
+		<sheet display="false">
+			<description>Your spellcasting ability for chosen cantrip is Wisdom..</description>
```
Comment (JayPea00101010) [inline unearthed-arcana/2021/20211008.xml:90]:
```suggestion
			<description>Your spellcasting ability for chosen cantrip is Wisdom.</description>
```

File: unearthed-arcana/2021/20211008.xml [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+			<stat name="ua:astral fire:spellcasting" value="Wisdom" inline="true" />
+		</rules>
+	</element>
+	<element name="Charisma" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_ASTRAL_FIRE_CHARISMA">
+		<supports>UA20211008 Astral Fire</supports>
+		<description>
+			<p>Your spellcasting ability for chosen cantrip is Charisma.</p>
+		</description>
+		<sheet display="false">
+			<description>Your spellcasting ability for chosen cantrip is Charisma..</description>
```
Comment (JayPea00101010) [inline unearthed-arcana/2021/20211008.xml:102]:
```suggestion
			<description>Your spellcasting ability for chosen cantrip is Charisma.</description>
```

**Review by AngryGuineaPig — APPROVED** [review]

File: unearthed-arcana/2021/20211008.xml [hunk 4]
```diff
@@ -0,0 +1,551 @@
...
+		</description>
+		<sheet usage="1/Long Rest">
+			<description>When you succeed on a death save, you can regain {{ua:radiant soul}} hit points.</description>
+		</sheet>
+		<rules>
+			<!-- <select type="Racial Trait" name="Ability Score (Radiant Soul)" supports="UA20211008 Radiant Soul" /> -->
+			<stat name="ua:radiant soul" value="proficiency" bonus="base" />
+			<stat name="ua:radiant soul" value="intelligence:modifier" bonus="ability" />
+			<stat name="ua:radiant soul" value="wisdom:modifier" bonus="ability" />
+			<stat name="ua:radiant soul" value="charisma:modifier" bonus="ability" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20211008.xml:142]: Based on the description, I believe that it should use the same the same stat as Astral Fire

File: unearthed-arcana/2021/20211008.xml [hunk 5]
```diff
@@ -0,0 +1,551 @@
...
+		<description>
+      		<p>Plasmoids are amorphous beings with no typical shape. In the presence of other folk, they often adopt a similar form, but there’s little chance of mistaking a plasmoid for anything else. They consume food by osmosis, the way an amoeba does, and excrete waste through tiny pores. They breathe by absorbing oxygen through another set of pores, and their limbs are strong and flexible enough to grasp and manipulate weapons and tools. Although most plasmoids are translucent gray, they can alter their color and translucence by absorbing dyes through their pores.</p>
+			<p class="indent">Plasmoids don’t have internal organs of the usual sort.</p>
+			<p class="indent">Their bodies are composed of cells, fibers, plasma-like ooze, and clusters of nerves called ganglia. These nerves enable a plasmoid to detect light, heat, texture, sound, pain, and vibrations. Plasmoids can stiffen the outer layers of their bodies to maintain a humanlike shape, so they can wear clothing and accessories. They speak by forcing air out of tubular cavities that constrict to produce sound.</p>
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20211008.xml:391]:
```suggestion
			<p class="indent">Plasmoids don’t have internal organs of the usual sort. Their bodies are composed of cells, fibers, plasma-like ooze, and clusters of nerves called ganglia. These nerves enable a plasmoid to detect light, heat, texture, sound, pain, and vibrations. Plasmoids can stiffen the outer layers of their bodies to maintain a humanlike shape, so they can wear clothing and accessories. They speak by forcing air out of tubular cavities that constrict to produce sound.</p>
```

**Pull request merged**